.gitignore

```diff
@@ -68,5 +68,9 @@ bin/
 # Gradle files
 .gradle/
 
+# node files
+node/
+node_modules/
```

Comment:
Perhaps you should use a more specific path for these.  Or you could put an extra .gitignore into the relevant parent directory.